### DISCUSSION TITLE
Add favorite menu item changes to Remove favorite as part of tap

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44012] [Mobile] [answered]
### Select Topic Area

Bug

### Body

1. Open GitHub Mobile
2. Tap 🔍
3. Type a search (for a repository)
4. Tap `Repositories with "..."`
5. Tap a desired repository
6. Tap ⋮
7. Tap `Add favorite` (or `Remove favorite`)
8. See that the menu item changes to `Remove favorite` (or `Add favorite`) before closing

### Expected results

The menu item should close first and then its value should be updated.

**Comment by HenSquared** [comment]
Hey @jsoref Thanks for all the feedback! 

What device are you using? Is that consistent for all the feedback items you opened?

**Reply by jsoref** [reply]
Yes, I'll be using the same Google Pixel 3A / Android 12

**Reply by HenSquared** [reply]
Thanks!

**Comment by HenSquared** [comment, marked as answer]
Thanks for the feedback. I've opened an issue to get this fixed!

**Comment by ghost** [comment]
> Hey @jsoref Thanks for all the feedback! 
> 
> What device are you using? Is that consistent for all the feedback items you opened? 
>
